safechef/safechef-v1.context.jsonld

```diff
@@ -0,0 +1,30 @@
+{
+  "@context": {
+    "@version": 1.1,
```

Comment:
```suggestion
```

No need for `@version` anymore.

Reply:
No? Standard is now 1.1 for VC processing?